### ISSUE TITLE
System.TypeInitializationException:“The type initializer for 'fashion_mnist' threw an exception.”

### ISSUE THREAD

**Issue by typetrade** [losttech/Gradient-Samples#14]
PS D:\github> git clone https://github.com/losttech/Gradient-Samples.git
Cloning into 'Gradient-Samples'...
remote: Enumerating objects: 1765, done.
remote: Counting objects: 100% (282/282), done.
remote: Compressing objects: 100% (64/64), done.
remote: Total 1765 (delta 253), reused 224 (delta 214), pack-reused 1483Receiving objects:  99% (1748/1765)
Receiving objects: 100% (1765/1765), 690.92 KiB | 1.52 MiB/s, done.
Resolving deltas: 100% (1176/1176), done.

Open FashionMnistClassification.csproj and run

System.TypeInitializationException
  HResult=0x80131534
  Message=The type initializer for 'fashion_mnist' threw an exception.
  Source=LostTech.TensorFlow
  StackTrace:
   在 tensorflow.tf.keras.datasets.fashion_mnist.load_data()
   在 LostTech.Gradient.Samples.FashionMnistClassification.Main() 在 D:\Github\Gradient-Samples\FashionMnistClassification\FashionMnistClassification.cs 中: 第 17 行

  此异常最初是在此调用堆栈中引发的: 
    [外部代码]

内部异常 1:
GradientInitializationException: None of discovered matching Python environments have TensorFlow 2.* installed. Please, install tensorflow or tensorflow-gpu 2.5.*, or use GradientEngine to configure Python home to use a different environment.

Found Python:
3.10-X64 at C:\Python310\

(dynamic train, dynamic test) = tf.keras.datasets.fashion_mnist.load_data();

Q: Where is the data of fashion_mnist?

**Comment by lostmsu** [comment]
It says in the error that tensorflow-gpu 2.5 must be installed.

Using your python run
```powershell
python -m pip install tensorflow-gpu==2.5.*
```
